sdk/eventhub/Azure.Messaging.EventHubs/src/ApiCompatBaseline.txt

```diff
@@ -1,2 +0,0 @@
-# Baselining these until the next GA. This is not a compatibility issue; the impacted member was inappropriately tagged as non-visible and provides value for customers being discoverable.
```

Comment:
Interesting.. is this not needed anymore?

Reply:
I'm hoping not.  It was supposed to just be needed for 1-2 versions and it's been hanging around for a year+.